query_tests/generate_rnd_queries.py

```diff
@@ -442,7 +469,26 @@ def generate_query(data_faker, columns, schema, table):
             expressions.append(random.choice(CONJUNCTIONS))
         expressions.append(expr)
     filter_ = ' '.join(expressions)
-    return f'SELECT count(*) FROM "{schema}"."{table}" WHERE {filter_};'
+    use_window_functions = every(20)
+    wfunction_expr = None
+    if use_window_functions:
+        wfunction = random.choice(WINDOW_FUNCTIONS)
+        if wfunction:
```

Comment:
Is this `if` required? `random.choice` afaik always returns one item of the list? (and it is not empty)

Reply:
python uber defensive development :)